server/routes/upload.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const router = express.Router();
+const { uploadImage } = require("../controllers/upload");
+
+router.route("/").post(uploadImage);
```

Comment:
We won't require this route, please see above

Reply:
added new function to profile.js in controllers and updated the profile.js in routes accordingly